.github/styles/Vocab/Docs/accept.txt

```diff
@@ -8,6 +8,7 @@ Avro
 [Bb]oolean
 [Bb]oot
 [Bb]usiness
+[Cc]odecs
```

Comment:
Are you sure we need this? I don't see we use "Codecs" with capital "C" anywhere

Reply:
Not at the moment, but I just figured that it might be used for example at the beginning of a sentence at some stage, so the capitalized spelling might as well be allowed too ;)

Reply:
Ah, I think I see what you mean. Is "codecs" not a word from a normal vocabulary? You had to add "codecs", right? and [Cc] is just on top of it

Reply:
Yep :)

Reply:
done